src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -2,157 +2,161 @@
+
+        public static readonly Option LoggerOption = new Option<IEnumerable<string>>(new string[] { "-l", "--logger" }, LocalizableStrings.CmdLoggerDescription)
+        {
+            Argument = new Argument<IEnumerable<string>>(LocalizableStrings.CmdLoggerOption)
+            {
+                Arity = ArgumentArity.OneOrMore
+            }
+        }.ForwardAsSingle(o => {
+            var loggersString = string.Join(";", GetSemiColonEscapedArgs(o));
+
+            return $"-property:VSTestLogger=\"{loggersString}\"";
+        });
+
+        public static readonly Option OutputOption = new Option<string>(new string[] { "-o", "--output" }, LocalizableStrings.CmdOutputDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdOutputDir)
+        }.ForwardAsSingle(o => $"-property:OutputPath={CommandDirectoryContext.GetFullPath(o)}");
+
+        public static readonly Option DiagOption = new Option<string>(new string[] { "-d", "--diag" }, LocalizableStrings.CmdPathTologFileDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdPathToLogFile)
+        }.ForwardAsSingle(o => $"-property:VSTestDiag={CommandDirectoryContext.GetFullPath(o)}");
+
+        public static readonly Option NoBuildOption = new Option<bool>("--no-build", LocalizableStrings.CmdNoBuildDescription)
+            .ForwardAs("-property:VSTestNoBuild=true");
+
+        public static readonly Option ResultsOption = new Option<string>(new string[] { "-r", "--results-directory" }, LocalizableStrings.CmdResultsDirectoryDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdPathToResultsDirectory)
+        }.ForwardAsSingle(o => $"-property:VSTestResultsDirectory={CommandDirectoryContext.GetFullPath(o)}");
+
+        public static readonly Option CollectOption = new Option<IEnumerable<string>>("--collect", LocalizableStrings.cmdCollectDescription)
+        {
+            Argument = new Argument<IEnumerable<string>>(LocalizableStrings.cmdCollectFriendlyName)
+            {
+                Arity = ArgumentArity.OneOrMore
+            }
+        }.ForwardAsSingle(o => $"-property:VSTestCollect=\"{string.Join(";", o)}\"");
+
+        public static readonly Option BlameOption = new Option<bool>("--blame", LocalizableStrings.CmdBlameDescription)
+            .ForwardAs("-property:VSTestBlame=true");
+
+        public static readonly Option BlameCrashOption = new Option<bool>("--blame-crash", LocalizableStrings.CmdBlameCrashDescription)
+            .ForwardAs("-property:VSTestBlameCrash=true");
+
+        public static readonly Argument BlameCrashDumpArgument = new Argument<string>(LocalizableStrings.CrashDumpTypeArgumentName).FromAmong(new string[] { "full", "mini" });
+
+        public static readonly Option BlameCrashDumpOption = new Option<string>("--blame-crash-dump-type", LocalizableStrings.CmdBlameCrashDumpTypeDescription)
```

Comment:
There is no equivalent of 

```
Accept.AnyOneOf(
                            "full",
                            "mini")
``` ?

Reply:
There is, FromAmong, which I'm using with the options above and below this one. Should this option only accept full or mini as well? The original code doesn't specify that.